sps30-i2c/sps30.c

```diff
@@ -215,10 +224,12 @@ int16_t sps30_start_manual_fan_cleaning() {
     if (ret)
         return ret;
 
-    sensirion_sleep_usec(SPS_WRITE_DELAY_US);
+    sensirion_sleep_usec(SPS_CMD_DELAY_USEC);
     return 0;
 }
 
 int16_t sps30_reset() {
-    return sensirion_i2c_write_cmd(SPS30_I2C_ADDRESS, SPS_CMD_RESET);
+    int16_t ret = sensirion_i2c_write_cmd(SPS30_I2C_ADDRESS, SPS_CMD_RESET);
+    sensirion_sleep_usec(SPS_RESET_DELAY_USEC);
```

Comment:
The reset sleep should happen with the caller since it's typically much larger than the others